flink-libraries/flink-table/src/test/java/org/apache/flink/table/runtime/utils/JavaUserDefinedTableFunctions.java

```diff
@@ -35,4 +35,15 @@ public void eval(Integer a, Long b, Long c) {
 			collect(c);
 		}
 	}
+
+	/**
+	 * Emit every input string.
+	 */
+	public static class JavaVarsArgTableFunc0 extends TableFunction<String> {
+		public void eval(String... strs) {
+			for (String s : strs) {
+				collect(s);
+			}
+		}
```

Comment:
Add an other eval for method match check.(only a suggestion).
{code}
public void eval(int ival, String sVal) {
			while (ival-- > 0) {
				collect(sVal);
			}
		}
{code}